File: 백준/Silver/1929. 소수 구하기/소수 구하기.py
import math
input = __import__('sys').stdin.readline

M, N = map(int, input().split())

for i in range(M, N+1):
    check = True
    if i == 1:
        continue
    for j in range(2, int(math.sqrt(i))+1):
        if i%j == 0:
            check = False
            break
    if check:
        print(i)

File: 백준/Silver/1929. 소수 구하기/소수 구하기.cc
#include <iostream>
#include <cmath>
using namespace std;

int main() {
    int M, N;
    cin >> M >> N;
    for (int i=M; i<=N; i++) {
        if (i == 1) continue;
        int check = true;
        for (int j=2; j<(int)sqrt(i) + 1; j++) {
            if (i%j == 0) {
                check = false;
                break;
            }
        }
        if (check) cout << i << endl;
    }
    return 0;
}